pipelines/LSSTCam/Isr-cal.yaml

```diff
@@ -10,6 +10,7 @@ tasks:
   isr:
     class: lsst.ip.isr.IsrTaskLSST
     config:
+      ampNoiseThreshold: .nan
```

Comment:
I'm surprised this works...

Is the "." needed?

Reply:
Without the "." it's taken as a `str` 
```TypeError: Value nan is of incorrect type str. Expected type float```